about.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>about page</title>
+    <link rel="stylesheet" href="./styles/about.css">
+    <link rel="stylesheet" href="./styles/index.css">
```

Comment:
- [ ] Make sure that both pages have the same footer as requested by the previous reviewer.